packages/agora-rtc-react/src/hooks/tracks.ts

```diff
@@ -349,3 +355,109 @@ export function useRemoteVideoTracks(
   }, [isConnected, resolvedClient, users]);
   return tracks;
 }
+
+/**
+ * a hook can create a local video track
+ * unpublish track on unmount.
+ */
+export function useLocalVideoTrack(
```

Comment:
Does it mean "Local Video" = "Camera"?

Reply:
Yes, `Camera` is better?
`useLocalVideoTrack` → `useLocalCameraTrack`

Reply:
I guess so.

Reply:
fixed